examples/miral-shell/miral-app.sh

```diff
@@ -77,8 +77,14 @@ unset QT_QPA_PLATFORMTHEME
 
 if [ "${miral_server}" == "miral-shell" ]
 then
+  if [ "$(lsb_release -c -s)" == "xenial" ]
+  then
+    export MIR_SERVER_APP_ENV=GDK_BACKEND=x11:QT_QPA_PLATFORM=ubuntumirclient:SDL_VIDEODRIVER=mir:NO_AT_BRIDGE=1
```

Comment:
Wrapping the value of `MIR_SERVER_APP_ENV` in quotes may make it more obvious what is being assigned to what .

Reply:
Done